doc/Caliptra.md

```diff
@@ -34,19 +34,21 @@ To drive agility of specification definition and to maximize applicability, the
 
 Enhancements and advanced use cases and applications are outside the scope of this specification and may be developed in the form of a roadmap for the Silicon RoT and community engagement.
 
-Caliptra defines a design standard for a Silicon internal RoT baseline. This standard satisfies a Root of Trust for Measurement (RTM) role. The open-source implementation of Caliptra drives transparency into the RTM and measurement mechanism that anchor hardware attestation. The SoC must measure the code and configuration it boots into Caliptra. Caliptra must store these measurements and report them with signed attestations rooted in unique per-asset cryptographic entropy. As such, Caliptra serves as a Root of Trust for Identity (RTI) for the SoC.
+Caliptra 2.0 defines a design standard for a Silicon internal RoT baseline.  This standard satisfies a Root of Trust for Measurement (RTM) and cryptographic services for the SoC. The SoC must measure the code and configuration it boots into Caliptra in this configuration. Caliptra must store these measurements and report them with signed attestations rooted in unique per-asset cryptographic entropy. As such, Caliptra serves as a Root of Trust for Identity (RTI) for the SoC.
```

Comment:
> Caliptra 2.0

At the top of this doc we say this is Revision 2.0, Version 0.5. Should we update?

Reply:
Are you saying not call 0.5 yet?

Reply:
Ah sorry, I got confused with Revision vs Version. Disregard.